docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/cookiecutter.json

```diff
@@ -0,0 +1,5 @@
+{
+  "project_name": "Project Name",
+  "project_slug": "Project Slug",
+  "project_site_name": "Site Name"
+}
```

Comment:
You can delete this file – it doesn't need to be duplicated in the app directory, only in the root directory of the template.